packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/DepositMethods/template.success.tsx

```diff
@@ -67,7 +67,7 @@ const getType = (value: BSPaymentMethodType) => {
       )
     case BSPaymentTypes.BANK_ACCOUNT:
       let text
-      if (value.currency === 'EUR' || value.currency === 'GBP') {
+      if (value.currency === 'EUR' || value.currency === 'GBP' || value.currency === 'ARS') {
```

Comment:
ARS appears to be wire transfer only according the code i previously reviewed...?